site/src/examples/splitter/MultiOrientational.tsx

```diff
@@ -1,5 +1,10 @@
-import { FlexLayout, Text } from "@salt-ds/core";
-import { SplitHandle, SplitPanel, Splitter } from "@salt-ds/lab";
+import {
+  FlexLayout,
+  SplitHandle,
+  SplitPanel,
+  Splitter,
+  Text,
+} from "@salt-ds/core";
 
 import styles from "./splitter.module.css";
```

Comment:
in splitter.module.css we have 
/* stylelint-disable salt/correct-theme-token-usage */

this is the only example in core that has this ?

Reply:
Done!